ledger/caching/src/main/scala/com/daml/caching/DropwizardStatsCounter.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.caching
+
+import com.daml.metrics.CacheMetrics
+import com.github.benmanes.caffeine.cache.stats.{CacheStats, StatsCounter}
+
+private[caching] final class DropwizardStatsCounter(
+    metrics: CacheMetrics,
+) extends StatsCounter {
+
+  override def recordHits(i: Int): Unit =
+    metrics.hitCount.inc(i.toLong)
+
+  override def recordMisses(i: Int): Unit =
+    metrics.missCount.inc(i.toLong)
+
+  override def recordLoadSuccess(l: Long): Unit =
+    metrics.loadSuccessCount.inc(l)
```

Comment:
You only need to increment it by one.
```suggestion
    metrics.loadSuccessCount.inc()
```

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6043/commits/ded4b4a0c1492bafb03a5edd76c7e7a682c9de9b